### PR TITLE
Slepos tests

### PR DESCRIPTION
**Pull request by psladek**

### CODE REVIEW

**Comment by okurz** [comment]
For new test modules you need to add a Summary and Maintainer information. For an example see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/remote/remote_target.pm#L11

The commit message could have a bit more details, e.g. what issue is fixed, why this needs to change, to which versions of which product it applies, link to a bug or a fate entry, the choices you made, etc. Also see http://chris.beams.io/posts/git-commit/ as a helpful guide how to write good commit messages.

File: tests/slepos/check_services.pm [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "basetest";
```
Comment (aaannz) [inline tests/slepos/check_services.pm]: I'm missing `use strict;` (or `use 5.012` or similar) and  `use warnings;`

**Comment by psladek** [comment]
"use" declarations are now fixed.

**Comment by okurz** [comment]
* please start the subject lines of each commit in imperative mode, i.e. "Add services checking…"
* call `tools/tidy` to fix the failing check

**Comment by okurz** [comment]
Please provide a link or screenshot to a local verification run.

File: tests/slepos/build_images_kiwi.pm [hunk 2]
```diff
@@ -7,29 +7,55 @@
...
+        dwl_image($im_hr, $target);
+    }
+    else {
+        build_image($target, $template, $linux32, $mod);
+    }
+}
+
+
+
+sub dwl_image {
```
Comment (okurz) [inline tests/slepos/build_images_kiwi.pm]: oh, *that's* what `dwl` means ;-) Yes, please change to `download_image`

**Comment by psladek** [comment]
fixed test description
subroutine name clarified

**Comment by aaannz** [comment]
@sysrich any other remarks?

**Comment by okurz** [comment]
Please provide a link or screenshot to a local verification run.

Also commit message subject lines should not end with a dot.

**Comment by nadvornik** [comment]
@okurz what should be verified by the verification run? The new SLEPOS tests or that it did not break the SLES tests? I think the later is enough. 
For now the SLEPOS tests are in development phase and are running only on our local machines so the policy does not have be that strict.

**Comment by okurz** [comment]
> what should be verified by the verification run? The new SLEPOS tests or that it did not break the SLES tests? I think the later is enough.

yeah, sure. That's enough.

> For now the SLEPOS tests are in development phase and are running only on our local machines so the policy does not have be that strict.

Of course. I trust you on the workflow for SLEPOS. I can see that no non-SLEPOS code is touched and no checks failed so we can also merge it right away after the commits look much cleaner.

@psladek, @nadvornik thank you.

**Pull request merged**